pages/advanced/passkeys/tutorials/react-native.mdx

```diff
@@ -0,0 +1,266 @@
+
+![react-native-passkeys-app-3.png](../../../../assets/react-native-passkeys-app-3.png)
+
+## 7. Setup and host `AASA` and `assetslinks.json` files
+
+To use passkeys in the app, you need to complete some additional setup in the Android and iOS projects. To facilitate this, we are providing a simple [Node Express web server](https://github.com/5afe/aasa-server) for testing.
+
+Clone and install the repository with:
+
+```bash
+git clone https://github.com/5afe/aasa-server.git
+```
+
+Fill in your data in the `apple-app-site-association` and `assetlinks.json` files as follows:
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['iOS', 'Android']}>
+  <Tabs.Tab>
+    - Login or subscribe to the Apple Developer Program and [locate your Team
+    ID](https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/).
+    - Add your Team ID instead of `add_your_apple_team_id_here` in the
+    `apple-app-site-association` file in the AASA server.
+    - Add your provisioning profile to sign the app through Xcode. You can open the generated iOS project in Xcode (open the folder `/ios` in your project) and
+    under `Signing & Capabilities` choose your team. Make sure that the `Bundle Identifier` is `com.safeglobal.reactnativepasskeys`.
+    - Activate FaceId. Go to Features > Face ID > Enrolled in the iOS simulator menu.
+  </Tabs.Tab>
+  <Tabs.Tab>
+    - Add a Google account in the settings of the Android emulated phone.
+    - Get an Android keystore SHA 256 fingerprint with this command (from the root
+    of your project):
+```bash
+keytool -list -v -keystore ./android/app/debug.keystore -alias androiddebugkey -storepass android -keypass android
+```
+    - Copy the value in front of `SHA 256` at the place of
+    `add_your_sha256_cert_fingerprints_here` in the assetslinks.json file.
+  </Tabs.Tab>
+</Tabs>
+
+{/* <!-- vale on --> */}
+
+You can now start the server with:
+
+```bash
+cd aasa-server
+npm install
+npm start
+```
+
+Once you have it running, you need a public domain secured with SSL to test the passkeys (`localhost` is not valid). You can use a service like [ngrok](https://download.ngrok.com/mac-os) to create a tunnel to your local server.
+
+After the installation, create a tunnel to the localhost port you are using, which should be `3000` if you are using the provided server.
+
+```bash
+ngrok http 3000
+```
+
+The ngrok service will provide you with a public domain. You must copy this domain (without `https://`) and add it to your project files after completing the steps below:
+
+- [passkeys.ts](https://github.com/5afe/react-native-passkeys-tutorial/blob/913cc17f353062bf1c4c55ad303e8623b6035a6b/lib/passkeys.ts#L8)
+- [app.json](https://github.com/5afe/react-native-passkeys-tutorial/blob/913cc17f353062bf1c4c55ad303e8623b6035a6b/app.json#L19). It's important to have the [developer mode](https://developer.apple.com/documentation/xcode/configuring-an-associated-domain) activated (?mode=developer) in your `app.json`.
```

Comment:
Better to update to add the three lines https://github.com/5afe/react-native-passkeys-tutorial/blob/913cc17f353062bf1c4c55ad303e8623b6035a6b/app.json#L18-L20